internal/daemon/controller/handlers/targets/target_service_test.go

```diff
@@ -0,0 +1,74 @@
+package targets
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/boundary/internal/servers"
+	"github.com/hashicorp/boundary/internal/types/scope"
+	pb "github.com/hashicorp/boundary/sdk/pbs/controller/api/resources/targets"
+	"github.com/hashicorp/go-bexpr"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
any chance you'd take a stab at implementing a basic unit test for `AuthorizeSession(...)`?

Reply:
ty!